src/app/backend/replicasetpodsmetrics.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+
+	"encoding/json"
+	"fmt"
+	heapster "k8s.io/heapster/api/v1/types"
+	"k8s.io/kubernetes/pkg/api"
+	unversioned "k8s.io/kubernetes/pkg/client/unversioned"
+	"strings"
+)
+
+const (
+	cpuUsage                 = "cpu-usage"
+	memoryUsage              = "memory-usage"
+	megaByte                 = 1024 * 1024
+	defaultHeapsterNamespace = "kube-system"
+	defaultHeapsterScheme    = "http"
+	defaultHeapsterService   = "heapster"
+	defaultHeapsterPort      = "" // use the first exposed port on the service
+)
+
+// Metrics map by pod name.
+type ReplicaSetMetricsByPod struct {
+	// Metrics map by pod name
+	MetricsMap map[string]PodMetrics `json:"metricsMap"`
+}
+
+// Pod metrics structure.
+type PodMetrics struct {
+	// Cumulative CPU usage on all cores in nanoseconds.
```

Comment:
Hmm... The return value should be something directly display-able in the UI. For example, milicores.